app/pages/editorg/contextCreator.js

```diff
@@ -0,0 +1,16 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../config';
+
+export const getContext = ({ organisation }) => ({
+  ...manifest,
+  organisationName: organisation.name,
+  odsCode: organisation.odsCode,
+  primaryRoleId: organisation.primaryRoleId,
+  address: organisation.address ? Object.keys(organisation.address).reduce((acc, key) => {
```

Comment:
Would using Object.values make this any simpler? Since we are only really interested in the values of the object.